src/main/java/io/cdap/e2e/utils/PageHelper.java

```diff
@@ -43,4 +51,33 @@ public static void dismissAlertIfPresent() {
       logger.info("No alert present");
     }
   }
+
+  /**
+   * Switch to browser window based on the window index
+   *
+   * @param windowIndex Starts with 0
+   */
+  public static void switchToWindow(int windowIndex) {
+    WaitHelper.waitForNewWindow(windowIndex + 1);
+    logger.info("Switching to Window at index (index starts with 0): " + windowIndex);
+    ArrayList<String> listOfWindows = new ArrayList<>(SeleniumDriver.getDriver().getWindowHandles());
+    SeleniumDriver.getDriver().switchTo().window(listOfWindows.get(windowIndex));
+  }
+
+  /**
+   * Close the current window
+   */
+  public static void closeCurrentWindow() {
+    logger.info("Close the current window");
+    SeleniumDriver.getDriver().close();
+  }
+
+  /**
+   * Switch back to main window (index: 0)
+   */
+  public static void switchBackToMainWindow() {
```

Comment:
The main window here would be the one specified in `cdfurl` in `connectionparameters` file, right?
Can we add this information as a comment?

Reply:
I see 👍